Tests/MongoSwiftTests/CrudTests.swift

```diff
@@ -401,8 +405,11 @@ private class DistinctTest: CrudTest {
         let resultDoc: Document = ["result": try coll.distinct(fieldName: fieldName,
                                                                filter: filter ?? [:],
                                                                options: options)]
-        let expectedDoc: Document = ["result": self.result]
-        expect(resultDoc).to(equal(expectedDoc))
+        let expectedDoc: Document
+        if let result = self.result {
+            expectedDoc = ["result": result]
```

Comment:
Should this be rewritten as `let expectedDoc: Document = ["result": result]`? There's probably no longer a reason to declare `expectedDoc` outside of this conditional.

Reply:
Yup, done.